libros/packages/print_options_list.py

```diff
@@ -8,18 +13,19 @@
 """
 
 options = [
```

Comment:
As mentioned in Slack:

If you make this a Dict[Tuple], keyed by `1-5`, then you could add the execute function to the tuple:

```
options = {
1: ("Find a book", find_book),
2: ("Loan out a book, loan_book(scan_barcode()))
```

You could then use this to replace the `if/elif` blocks

Reply:
I really like that idea, I'll definitely do that.